README.md

```diff
@@ -1 +1,59 @@
-# aws-centralized-monitoring
\ No newline at end of file
+# aws-centralized-monitoring
+
+## Table of Contents
+- [Instructions](#instructions)
+- [Support](#support)
+
+## Instructions
+### Why use this script?
+The primary reason to use this script is to enable centralized, multi-account monitoring in AWS. You may choose to do this via the AWS console, but the process involves multiple steps that can be prone to human error. Additionally, this process must be repeated in each AWS region you wish to collect metrics in. This script will automate the process and deploy the necessary resources in each region you specify (or all AWS default regions) for a quick and seamless onboarding to AWS Centralized CloudWatch Monitoring.
+
+### Alternative ways to achieve multi-account monitoring (and why this is better)
+There are multiple ways to achieve centralized monitoring in AWS. If you do not need to use the CloudWatch APIs, you can set up a
+multi-account dashboard in the CloudWatch console, but you'll have to manually select and repeat this process for relevant metrics and resources. The advantage to using this script is that it will enable you to centralize, collect and report on any (standard) AWS metric for any resource you have access to.
+
+### Ongoing maintenance
+By leveraging CloudFormation StackSets and AWS Organizations you greatly reduce your ongoing burden to maintain centralized monitoring. As you add and delete accounts in your organization, new Stacks creating links to your centralized sink will automatically be created and destroyed on a per-region, per-account basis.
+
+### Prerequisites:
+- [`Python3`](https://www.python.org/downloads/) (We're using 3.11)
+- `Pip3` (Comes standard with Python > 3.4)
+- Optional, but highly recommended `VirtualEnv`
+  - `pip3 install virtualenv`
+- AWS CLI Credentials for your management account
+
+### Setup your environment
+- If you have `virtualenv` installed:
+  - `source env_setup.sh`
+- If you don't have `virtualenv`:
+  - `pip3 install -r requirements.txt`
+
+### Run the script
+- `python3 create_links_sinks.py --sink-name SINK-NAME --regions "region1,region2,region3" --profile AWS-CLI-PROFILE --organization ORGANIZATION --organization-unit ORGANIZATION-UNIT --excluded-accounts "accountID1,accountID2,accountID3"`
+  - `sink-name`: Human-friendly name for your sink and StackSet
+  - `regions`: _Optional_ Double quoted, comma separated list of regions to create a sink and StackSet in
```

Comment:
imo it's unnecessary to specify that the regions arg should be double-quoted or to omit spaces, since you've already specified that the input should be a comma-separated list. The app never sees the double-quotes anyway and it's the user's responsibility to delimit arguments to the shell.

Anything that has spaces imo would just be malformed input, which could be handled in the usual way (e.g. if a user gave as a colon separated list or put other bad characters into the arg)